osu.Game/Screens/Menu/LogoVisualisation.cs

```diff
@@ -209,43 +202,40 @@ public override void Draw(Action<TexturedVertex2D> vertexAction)
                 ColourInfo colourInfo = DrawColourInfo.Colour;
                 colourInfo.ApplyChild(transparent_white);
 
-                if (audioData != null)
```

Comment:
How come we never got an inspection about this conditional if its readonly and already instantiated...

Reply:
Because NRT was turned off and this is the only null check in this file. R# is optimistic by default.